package.json

```diff
@@ -56,6 +56,9 @@
 		]
 	},
 	"prettier": {
+		"tabWidth": 4,
+		"useTabs": false,
+		"bracketSpacing": true,
```

Comment:
Ah sorry, I could have done this originally but I didn't realise there were still differences. 

@marinakalyuzhnaya if this is merged before changes from you, you can copy these changes and run `format` to avoid merge conflicts